### PR TITLE
Create COVID dashboard custom block.

### LINKED ISSUE #4005: Coronavirus Dashboard
As a coronavirus site owner, My Covid by the Numbers dashboard gets updated automatically every Monday, Wed, Fri mornings at X:XXAM.

Some points we discussed today in planning:
- We build through pairing/swarming. I'd rather move slower and make the investment that multiple people have a high comfort level in working on this in case we get [hit with chaos](https://thinkinsights.net/wp-content/uploads/Cynefin-Framework.png) in the future.
- The table will be updated for the public every M/W/F. It is unclear to me whether that is baked into the feed, or we need to control that flow.
- The table is accessible
-  The table (or surrounding page) indicates when the data contained therein was last updated.

Screenshot of dashboard info is here: https://osc-marketing.slack.com/archives/C0164FNN1PV/p1629392494356800

### PR DESCRIPTION
**Pull request by richardbporter**
Resolves #4005 

### Dev setup
- Block is placed on https://coronavirus.dev.drupal.uiowa.edu/covid-19-numbers

### Local setup
- Get the credentials from the [secrets file](https://docs.acquia.com/resource/secrets/#secrets-settings-php-file) on uiowa01 dev. Add them to the coronavirus site local settings file located at `docroot/sites/coronavirus.uiowa.edu/settings/local.settings.php`.
```
blt sync --site coronavirus.uiowa.edu &&
drush @coronavirus.local cim --source ../config/coronavirus.uiowa.edu --partial --yes &&
drush @coronavirus.local cim --yes
```
- Place the block on page via layout builder.

### Stories
- As a coronavirus.uiowa.edu site webmaster, I can place this block and understand how it functions.
- As a coronavirus.uiowa.edu site visitor, I can see automatic updates around 10am on M/W/F and previous reporting data on other dates.
- As a coronavirus.uiowa.edu site visitor using IE11, I can have the same experience as the story above.

### CODE REVIEW

**Comment by richardbporter** [comment]
0d781752ed46d1c1c878002b47afdacb2217aff1 seems to have broken things for me. Getting `InvalidArgumentException: Class "\Drupal\uiowa_covid\Controller\DataController" does not exist. in Drupal\Core\DependencyInjection\ClassResolver->getInstanceFromDefinition() (line 24 of core/lib/Drupal/Core/DependencyInjection/ClassResolver.php).` now... 😢

**Comment by richardbporter** [comment]
`vagrant reload --provision` fixed it. Something was cached somewhere.

**Comment by richardbporter** [comment]
Kinda want to rename the module to covid_data... :)

**Comment by joewhitsitt** [comment]
>Kinda want to rename the module to covid_data... :)

Is this something you are doing? Should it be moved back to draft?

**Comment by richardbporter** [comment]
> > Kinda want to rename the module to covid_data... :)
> 
> Is this something you are doing? Should it be moved back to draft?

Not doing it. We restored an old branch and thats what it was called. Local to site dir, so it might not make sense prefixed with uiowa_ but oh well.

**Comment by joewhitsitt** [comment]
pulling to test

**Review by joewhitsitt — APPROVED** [review]
Works as expected. Did not test against IE11.

**Comment by richardbporter** [comment]
Just realizing that using the secrets file will load the config overrides on every site. Not problematic but not ideal either. We could follow up with a settings form for this sites specifically but I think we should look into a more secure way to save that information first.

**Comment by dnkemper** [comment]
> Just realizing that using the secrets file will load the config overrides on every site. Not problematic but not ideal either. We could follow up with a settings form for this sites specifically but I think we should look into a more secure way to save that information first.

That's why Rick's hair is so big... it's full of secrets

**Comment by cory-skeers** [comment]
> That's why Rick's hair is so big... it's full of secrets

Dying!

File: config/coronavirus.uiowa.edu/config_split.config_split.site.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+uuid: 6f345119-3a28-4881-b397-5d25d337ea5c
+langcode: en
+status: true
+dependencies: {  }
+id: site
+label: Site
```
Comment (richardbporter) [inline config/coronavirus.uiowa.edu/config_split.config_split.site.yml:6]: The label could've/should've been named differently.

**Pull request merged**